src/components/elementView/elementTab.tsx

```diff
@@ -549,17 +553,9 @@ class ElementTab extends React.Component<ElementTabProps, ElementViewState> {
                 this.props.element !== ElementType.ASSET &&
                 this.props.changePlan
                   ? true
```

Comment:
This means you can't add a user or datacenter if you're in a change plan, is that desired?  I think that's inconsistent with Racks isn't it?  In that you can add a rack but there's a warning that it makes those changes live

Reply:
Also can you edit or delete datacenters, users, or models if you're in a change plan? I think it should be the same for all the write operations

Reply:
nope, this is expected behavior for change plan, and was reflects the logic we had for ev3. the only reason why you're allowed to add a rack in a change plan is theoretically you might need to place a change plan asset on a rack that doesn't exist now. we don't allow this anywhere else, which is why we only have a warning for racks